client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/unified-design-picker.tsx

```diff
@@ -392,7 +408,15 @@ const UnifiedDesignPickerStep: Step = ( { navigation, flow, stepName } ) => {
 	const isPluginBundleEligible = useIsPluginBundleEligible();
 	const isBundled = selectedDesign?.software_sets && selectedDesign.software_sets.length > 0;
 
+	const selectedDesignTier =
+		useSelector( ( state ) =>
+			selectedDesignThemeId ? getThemeTierForTheme( state, selectedDesignThemeId ) : null
+		)?.slug ?? FREE_THEME;
+
 	const isLockedTheme =
```

Comment:
Wow, this logical chain is very hard to grasp 😵

Reply:
Yeah, I think I can simplify it a bit.

Reply:
That would be great!

Reply:
My comment was for `isLockedTheme`, which was not really new to the PR. Sorry if it was confusing. Anyway, it can be addressed in the future.